awscrt/auth.py

```diff
@@ -306,6 +306,63 @@ def new_delegate(cls, get_credentials):
         binding = _awscrt.credentials_provider_new_delegate(get_credentials)
         return cls(binding)
 
+    @classmethod
+    def new_cognito(
+            cls,
+            endpoint,
+            identity,
+            tls_ctx,
+            logins=None,
+            custom_role_arn=None,
+            client_bootstrap=None,
+            http_proxy_options=None):
+        """
+        Creates a provider that sources credentials from the AWS Cognito Identity service.
+
+        Args:
+            endpoint (str): Cognito Identity service regional endpoint to source credentials from.
+
+            identity (str): Cognito identity to fetch credentials relative to.
+
+            tls_ctx (ClientTlsContext): Client TLS context to use when querying cognito credentials by HTTP.
+
+            logins (Optional[List[Tuple[str, str]]]): List of tuples specifying pairs of identity provider name and
+                token values, representing established login contexts for identity authentication purposes
```

Comment:
trivial: ~List~ -> `Sequence`

Some people like to use tuples instead of lists